deploy/evs-csi-plugin/kubernetes/csi-evs-driver.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: storage.k8s.io/v1
+kind: CSIDriver
+metadata:
+  name: evs.csi.openstack.org
```

Comment:
s/evs.csi.openstack.org/evs.csi.huaweicloud.org

Reply:
OK, done

Reply:
changed it in branch "feat-documents-yaml"